python/tchannel/messages/base.py

```diff
@@ -25,3 +30,59 @@ def __str__(self):
             str(self.__class__.__name__),
             ", ".join(attrs)
         )
+
+    @staticmethod
+    def compute_checksum(checksum_type, args, csum=0):
```

Comment:
I discussed this with Bryce. Bryce suggests I put here. Since only message will use checksum